nerves_defconfig

```diff
@@ -37,22 +37,13 @@ BR2_PACKAGE_LZ4=y
 BR2_PACKAGE_E2FSPROGS=y
 # BR2_PACKAGE_E2FSPROGS_FSCK is not set
 BR2_PACKAGE_DEJAVU=y
-BR2_PACKAGE_MESA3D=y
-BR2_PACKAGE_MESA3D_DRI_DRIVER_I915=y
-BR2_PACKAGE_MESA3D_DRI_DRIVER_I965=y
-BR2_PACKAGE_MESA3D_OPENGL_EGL=y
-BR2_PACKAGE_MESA3D_OPENGL_ES=y
 BR2_PACKAGE_QT5=y
 BR2_PACKAGE_QT5BASE_SQLITE_SYSTEM=y
-BR2_PACKAGE_QT5BASE_OPENGL_ES2=y
-BR2_PACKAGE_QT5BASE_LINUXFB=y
```

Comment:
Don't we need this one?

Reply:
I was also surprised by this one, but it seems that omitting this key enabled the default strategy which is to include `linuxfb`. This was a product of `savedefconfig`

Reply:
Ah ok, we’ll just need to remember this if we ever want to have linuxb and eglfs both available.